src/components/popup/details/details.jsx

```diff
@@ -45,30 +52,26 @@ export default class Details extends Component {
 	};
 
 	handleShowVendors = (filter) => {
-		this.state.vendors = this.getVendors(filter);
-		this.setState({
-			selectedPanelIndex: SECTION_VENDORS
-		});
+		this.state.vendors = this.filterVendors(filter);
```

Comment:
Jest jakiś powód, że ustawiasz stan w ten sposób? 
https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly

Reply:
No, there is no reason. Simply it is remaining from refactoring.